fosspay/app.py

```diff
@@ -89,3 +93,51 @@ def inject():
         'str': str,
         'int': int
     }
+
+# STRIPE CODE
+@app.route('/support/confirm_payment', methods=['POST'])
+def make_payment_intent():
+    # Creates a new PaymentIntent with items from the cart.
+    data = json.loads(request.data)
+    try:
+        amount = 500
+        currency = data['currency']
+        # START TODO refactor this code
```

Comment:
what kind of refactoring should be applied here? can it be applied within this same PR?